test/unit/messaging/process-return-message.test.js

```diff
@@ -1,25 +1,91 @@
 jest.mock('ffc-messaging')
+
 const processReturnMessage = require('../../../app/messaging/process-return-message')
```

Comment:
I would move this down underneath the mocks above it -- sometimes it is needed and won't work otherwise, this isn't the case here but if you do it this way you're less likely to come across that issue